elasticdl/docker/Dockerfile

```diff
@@ -4,6 +4,10 @@ ARG BASE_IMAGE
 FROM ${BASE_IMAGE} as dev
 ARG EXTRA_PYPI_INDEX=https://pypi.org/simple
 
+# Allows for log messages by `print` in Python to be immediately dumped
+# to the stream instead of being buffered.
+ENV PYTHONUNBUFFERED 0
```

Comment:
What's the side effect if the message is buffered? Will we lost any message?

Reply:
We cannot view the log message of `print` by `kubectl logs` and we will miss the log message if the pod is deleted.